src/prefabs/structures/FileUpload/index.ts

```diff
@@ -1,59 +1,41 @@
-import { component, toggle, variable } from '@betty-blocks/component-sdk';
-import { Button } from '../Button';
-import { options as defaultOptions } from './options/index';
-import { buttonOptions } from '..';
-import { updateOption } from '../../../utils';
+import { PrefabReference, component } from '@betty-blocks/component-sdk';
+import { fileUploadOptions } from './options';
+import { Configuration } from '../Configuration';
 
-interface Configuration {
-  supportImages?: boolean;
-  label?: string;
-}
-
-export const FileUpload = (config: Configuration = {}) => {
-  const options = defaultOptions(config.supportImages);
-
-  const categories = [
-    {
-      label: 'Validation',
-      expanded: false,
-      members: [
-        'required',
-        'hideDefaultError',
-        'accept',
-        'maxFileSize',
-        'maxFileSizeMessage',
-      ],
-    },
-    {
-      label: 'Styling',
-      expanded: false,
-      members: ['hideLabel', 'labelColor', 'helperColor', 'errorColor'],
-    },
-    {
-      label: 'Advanced settings',
-      expanded: false,
-      members: ['nameAttribute', 'dataComponentAttribute', 'actionVariableId'],
-    },
-  ];
+const defaultCategories = [
+  {
+    label: 'Validation',
+    expanded: false,
+    members: [
+      'required',
+      'hideDefaultError',
+      'accept',
+      'maxFileSize',
+      'maxFileSizeMessage',
+    ],
+  },
+  {
+    label: 'Styling',
+    expanded: false,
+    members: ['hideLabel', 'labelColor', 'helperColor', 'errorColor'],
+  },
+  {
+    label: 'Advanced settings',
+    expanded: false,
+    members: ['nameAttribute', 'dataComponentAttribute', 'actionVariableId'],
+  },
+];
 
-  if (config.supportImages) {
-    options.accept = updateOption(options.accept, {
-      value: ['image/*'],
-    });
-  }
+export const FileUpload = (
+  config: Configuration = {},
+  descendants: PrefabReference[] = [],
+) => {
+  const options = { ...(config.options || fileUploadOptions) };
```

Comment:
Shouldn't we apply config.options to the defaultOptions? So that we overwrite?

```javascript
{...fileUploadOptions, ...config.options}
````